target/linux/ramips/image/mt7621.mk

```diff
@@ -375,6 +375,15 @@ define Device/dlink_dir-878-a1
 endef
 TARGET_DEVICES += dlink_dir-878-a1
 
+define Device/dlink_dir-878-r1
+  $(Device/dlink_dir-8xx-r1)
+  DEVICE_MODEL := DIR-878
+  DEVICE_VARIANT := R1
+  DEVICE_PACKAGES += kmod-usb3 kmod-usb-ledtrig-usbport
```

Comment:
Is there any reason for including USB packages for the DIR-878 R1 considering the device doesn't have working USB ports (at least not without modding the hardware)?